.github/workflows/workflow_rerunner.yml

```diff
@@ -0,0 +1,48 @@
+name: Rerun Workflow with Known Errors
+
+on:
+  workflow_run:
+    workflows:
+      - Linux (Ubuntu 20.04, Python 3.11)
+      - Linux ARM64 (Ubuntu 20.04, Python 3.11)
+      - Linux Static CC (Ubuntu 22.04, Python 3.11, Clang)
+      - Linux RISC-V with Conan (Ubuntu 22.04, Python 3.10)
+      - Windows (VS 2019, Python 3.11)
+      - Windows Conditional Compilation (VS 2022, Python 3.11)
+    types:
+      - completed
+
+permissions:
+  actions: write
+  contents: read
+  statuses: read
+  checks: read
+
+jobs:
+  rerun:
+    name: Rerun Workflow
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
```

Comment:
We do not want to analyze logs of the successful pipelines. Unfortunately, the `on: workflow_run` GHA trigger [only has `completed` for type and not `failure`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow).